File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& arr) {

      if (arr.size() == 0){

      return 0;
      }
      else if(arr.size() == 1){

      return 1;
      }
      else{

       int v = arr[0];
       int count = 0;

       for(int i =0; i<arr.size() ; i++){
        if(arr[i] != v){
            v = arr[i];
            count = count +1;
            arr[count] = arr[i];
        }
       }

       return count+1;

      }

    }  
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        Set<Integer> s = new HashSet<>();

        for(int i =0; i< nums.length; i++){
                s.add(nums[i]);
        }
        
        List<Integer> temp = new ArrayList<>(s);

        Collections.sort(temp);
        int count =0;

        for(int i: temp){
            nums[count] = i;
            count++;
        }

        return s.size(); 

    }
}